script.js

```diff
@@ -61,20 +69,14 @@
 // Write your solution below.
 // =======================================================================
 
+var randomNumber = Math.floor(Math.random() * 10);
+var userNumber = prompt("Enter an Integer between 1 to 10");
 
+userNumber = parseInt(Math.floor(userNumber));
 
-
-
-
-
-
-
-
-
-
-
-
-
+if (userNumber == randomNumber) {
+  console.log("Your number matched the randomly generated one!");
+};
```

Comment:
Here, randomNumber actually goes from 0 to 9 because of the Math.floor function, e.g. if Math.random() gives 0.001, then Math.floor(0.001 * 10) gives 0. At the same time, Math.random() has a practically 0 chance of returning 1, which is needed in order for Math.floor to return 10 here.